packages/morph/lib/dom-helper.js

```diff
@@ -307,6 +307,7 @@ prototype.appendMorph = function(element, contextualElement) {
 };
 
 prototype.parseHTML = function(html, contextualElement) {
+  this.detectNamespace(contextualElement);
   var isSVGContent = (
     isSVG(this.namespace) &&
```

Comment:
Sure, that'd work.  We could even just have something like 
```
  if (interiorNamespace(contextualElement)) {
    return buildSVGDOM(html, this);
  } else {
    var nodes = buildHTMLDOM(html, contextualElement, this);
```
since `interiorNamespace` is doing the same check anyway.  Then `isSVG` would be unused and could be deleted.  Or we could write `isSVG(interiorNamespace(contextualElement)` if we want to be more explicit.